src/Raven.Client/Documents/Operations/Indexes/ResetIndexOperation.cs

```diff
@@ -10,35 +10,45 @@ namespace Raven.Client.Documents.Operations.Indexes
     public sealed class ResetIndexOperation : IMaintenanceOperation
     {
         private readonly string _indexName;
+        private readonly bool _asSideBySide;
 
         public ResetIndexOperation(string indexName)
         {
             _indexName = indexName ?? throw new ArgumentNullException(nameof(indexName));
         }
+        
+        public ResetIndexOperation(string indexName, bool asSideBySide)
```

Comment:
Maybe just `sideBySide`?

Reply:
Actually I would go with IndexResetMode, similar to what we have with IndexDeploymentMode